script.js

```diff
@@ -0,0 +1,137 @@
+const firstArrayCreate = () => {
+  const arrayLenght = +prompt(
+    "Specify the number of positions in the array by entering a number from 1 to 10"
+  );
+  const firstArray = [];
+  if (!Number.isInteger(arrayLenght)) {
+    console.log("Wrong value, please select another array lenght");
+    return "Wrong value, please select another array lenght";
+  }
+  for (let i = 1; i <= arrayLenght; i++) {
+    const userValue = +prompt(`Enter a numeric value №${i}`);
+    if (!Number.isInteger(userValue)) {
+      console.log("Wrong value, please select another argument");
+      i--;
+    } else {
+      firstArray.push(userValue);
+    }
+  }
+  // console.log(firstArray);
+};
+firstArrayCreate();
+
+const productsList = [
+  { productName: "bread", productPrice: 14, productQuantity: 2 },
+  { productName: "potatoes", productPrice: 32, productQuantity: 6 },
+  { productName: "tomato", productPrice: 51, productQuantity: 8 },
+  { productName: "garlic", productPrice: 12, productQuantity: 10 },
+  { productName: "apple", productPrice: 44, productQuantity: 7 },
+  { productName: "wattermelon", productPrice: 100, productQuantity: 1 },
+];
+
+let total = 0;
+productsList.forEach((el) => (total += el.productPrice * el.productQuantity));
+// console.log(`total - ${total}`);
+
+const minQuantity = productsList.sort(
+  (el1, el2) => el1.productQuantity - el2.productQuantity
+)[0];
+// console.log(
+//   `The mmin quantity ${minQuantity.productName}: ${minQuantity.productQuantity} items`
+// );
+
+let totalQuantity = 0;
+productsList.forEach((el) => (totalQuantity += el.productQuantity));
+// console.log(`total Quantity - ${totalQuantity}`);
+
+const maxPrice = productsList.sort(
+  (el1, el2) => el2.productPrice - el1.productPrice
+)[0];
+// console.log(
+//   `Most expensive product - ${maxPrice.productName}, it costs ${maxPrice.productPrice}`
+// );
+
+const error = "Wrong number of arguments, try again";
+const isValid = (x) => {
+  if (x !== undefined || x !== null) {
+    return x;
+  }
+};
+
+const addNewItem = () => {
+  const newItem = {
+    productName: "",
+    productPrice: 0,
+    productQuantity: 0,
+  };
+  for (let key in newItem) {
+    const userValue = prompt(key);
+
+    if (!isValid(userValue)) {
+      return console.log("error");
```

Comment:
```suggestion
      return console.log(error);
```
You have a specific value for it